osm2gtfs/tests/core/tests_creator_factory.py

```diff
@@ -0,0 +1,92 @@
+# coding=utf-8
+
+import unittest
+import os
+from osm2gtfs.core.configuration import Configuration
+
+
+class TestCoreCreatorFactory(unittest.TestCase):
+
+    def test_naming_convention(self):
+
+        path = os.path.dirname(__file__) + "/../../creators/"
+        creators = ['agency', 'feed_info', 'routes', 'schedule', 'stops', 'trips']
+
+        # pylint: disable=unused-variable
+        for subdir, dirs, files in os.walk(path):
+
+            # Loop through available directories of creators
+            for selector in dirs:
+
+                # Check if creator directory fits naming convention
+                self.assertTrue(
+                    self._check_naming_convention(selector),
+                    'The path "creators/' + selector +
+                    ' " doesn\'t fit the naming convention.')
+
+                # Check for existing configuration file
+                config_file_path = path + selector + "/" + selector + ".json"
+                self.assertTrue(
+                    os.path.isfile(config_file_path),
+                    'No correctly named configuration file found for: ' +
+                    selector)
+
+                # Check whether selector in config file is correctly
+                config = Configuration.load_config_file(open(config_file_path))
+                self.assertEqual(
+                    selector, config['selector'],
+                    "The selector (" + config['selector'] +
+                    ") in config file is not equal to the directory name: " +
+                    selector)
+
+                # Loop through available files in creator directories and check
+                # whether they are correctly named
+                for inner_subdir, inner_dirs, inner_files in os.walk(
+                        path + selector):
+
+                    for filename in inner_files:
+                        # Check the pyhton files (and ignore the others)
+                        if filename.endswith(
+                                '.py') and not filename == "__init__.py":
+
+                            # Check if selector has been properly applied to
+                            # file name
+                            self.assertTrue(
+                                filename.endswith(selector + '.py'),
+                                "This file is not well named: " + filename)
+
+                            # Check for valid creators
+                            split_filename = filename.split("_")
```

Comment:
This won't work for `feed_info` (in `creators` list above).

Reply:
Thanks for noticing!

Reply:
Looks a bit ugly, but I'm okay with it.

Reply:
That's okay for me! :+1: